extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -705,9 +705,15 @@ export class SafeframeHostApi {
   /**
    * Fires a delayed impression and notifies the Fluid creative that its
    * container has been resized.
+   * @param {number} newHeight The height expanded to.
    * @private
    */
-  onFluidResize_() {
+  onFluidResize_(newHeight) {
+    const iframe = dev().assertElement(this.baseInstance_.iframe);
+    const iframeHeight = parseInt(getStyle(iframe, 'height'), 10) || 0;
```

Comment:
are you sure getStyle returns an int for height, and not something like "100px" ?

Reply:
`getStyle` will return a string like in your example, but parseInt will do the right thing and give you back `100`.